pkg/config/odfi.go

```diff
@@ -245,6 +245,10 @@ type Storage struct {
 	// after downloading and processing of each file.
 	KeepRemoteFiles bool
 
+	// RemoveZeroByteFilesAfter is the amount of time we want to wait before deleting
+	// zero byte files.
+	RemoveZeroByteFilesAfter int
```

Comment:
Could we have this be a `time.Duration`? That way we'll be able to write `"10m"` or `"6h"` in the config file.

Reply:
you bet. I'll make that change.